authorization/invitation/invitation.go

```diff
@@ -0,0 +1,13 @@
+package invitation
+
+import (
+	uuid "github.com/satori/go.uuid"
+)
+
+type Invitation struct {
```

Comment:
Nice, I've done something similar too here https://github.com/fabric8-services/fabric8-auth/pull/347/files#diff-bd9af08e13589624d0629a20880b3c70 because of the difference between the entity in the db, and the response json.